web/src/utils/formatting.test.ts

```diff
@@ -23,27 +23,42 @@ describe('formatCo2', () => {
     const expected = '0 t';
     expect(actual).toBe(expected);
   });
+  it('uses same unit as another value would non-zero', () => {
+    const actual = formatCo2(230_500, 2_350_000);
+    const expected = '0.2 t';
+    expect(actual).toBe(expected);
+  });
 
   it('adds decimals if comparing with tons', () => {
-    const actual = formatCo2(12_003_500, 200_350_000);
+    const actual = formatCo2(200_500, 2_350_000);
     const expected = '0.2 t';
     expect(actual).toBe(expected);
   });
   it('handles values up to 100k', () => {
-    const actual = formatCo2(700_003_500_000);
-    const expected = '11,667 t';
+    const actual = formatCo2(99_000_000_000_000_000);
+    const expected = '99 Gt';
     expect(actual).toBe(expected);
   });
```

Comment:
I'd like to keep a test for values up to 100k to test the thousand separator, was this changed to that high on purpose?